### DISCUSSION TITLE
Port of MicroPython to STM32H750B-DK board ?

### DISCUSSION THREAD

**Discussion by LAPSyLAB** [micropython/micropython#10219] [STM32 / Pyboard]
Hello,

has anyone tried or has more info about port to STM32H750B-DK board ?
One limitation is probably Internal Flash of only 128KB, but plenty of QSPI Flash, so this shouldn't be a problem.
I've also heard about the problem with clock configuration - not sure if this one is true.

There is a port for STM32H7B3I_DK which seems to be the closest one. Has anyone tried or has advice how to approach port for STM32H750B-DK board ?

Thanks in advance,
regards.

**Comment by jahr** [comment]
Hello, some time ago I've written definitions for some H750 boards. Not the STM32H750B-DK but you may find how I dit it ;-) Check the articke [here](https://forum.micropython.org/viewtopic.php?f=12&t=11110&p=60964&hilit=h750#p60964) and (somehow outdated, but previously working ;-) ) definitions [here](https://github.com/jahr/FK750M3) and [here](https://github.com/jahr/WEACT_H750).

**Reply by LAPSyLAB** [reply]
Thanks for info, will take a look. How did you compile micropython for those boards - CubeIDE ?

thanks.

**Reply by jahr** [reply]
No, I've never used CubeIDE for building MP. You may put the above mentioned definitions to ports/stm32/boards/ directory (e.g. clone it there) and then build it in the standard way, i.e. from ports/stm32/ directory run e.g. `make BOARD=WEACT_H750`. But keep in mind that these definitions were tested more than one year ago, so it is possible that the build may fail.

**Comment by LAPSyLAB** [comment]
Hello,

 

Thanks for all the info and help. I think it will be much easier to start our port.

 

We will let you know if definitions compile…

 

Regards,

Robert.
